docs/securing_fledge.rst

```diff
@@ -349,23 +349,32 @@ Default ca certificate is available inside $FLEDGE_DATA/etc/certs and named as c
 
 Below are the steps to create custom certificate along with existing fledge based ca signed for auth certificates.
 
-a) Create a new certificate for username. Let say **test**
+**Using cURL**
 
-.. code-block:: console
+- To create a user with a unique name and case-insensitive. The cURL commands are provided below.
```

Comment:
The "and case-insensitive" does not read well. It maybe better to say that user names must be unique and are case insensitive in a note to draw attention this requirement.